### ISSUE TITLE
Doesn't work w/ ministat since???

### ISSUE THREAD

**Issue by cjw6k** [cjw6k/mf2bench#7]
```
ministat: invalid option -- 'A'
Unknown option
Usage: ministat [-C column] [-c confidence] [-d delimiter(s)] [-ns] [-w width] [file [file ...]]
        confidence = {80%, 90%, 95%, 98%, 99%, 99.5%}
        -C : column number to extract (starts and defaults to 1)
        -d : delimiter(s) string, default to " \t"
        -n : print summary statistics only, no graph/test
        -s : print avg/median/stddev bars on separate lines
        -w : width of graph/test output (default 74 or terminal width)
```

**Comment by cjw6k** [comment]
Delete the -A.

**Comment by cjw6k** [comment]
Fixed by #8
